designs/virtualization/4.6/Add-os-image-to-common-template/add-os-image-to-common-template 4.6.md

```diff
@@ -0,0 +1,56 @@
+---
+parent: Virtualization
+version: 4.6
+---
+# Add an operating system image to a common template
+
+## About
+
+We strive to provide an off-the-shelf OS image with every common template.
+This an ongoing effort.
+Hopefully this will provide the user with a much easier experience to creating a virtual machine.
+This OS image takes the shape of a disk image(data-volume) and can be attached to the OS template (one of the parameters within the common templates).
+These OS images, and common templates in general, won’t be editable for the user.
+Today, we don’t have OS images, so we need a temporary solution; offering the user a way to attach a new image that he adds to a common template, making it available on selecting that OS template field.
+
+
+## Adding an OS image
+
+![Create VM wizard - default](img/Admin-create-4.6-0-0.jpg)
+
+On clicking "Create VM" the user lands on the wizard.
+The user enters a VM name and clicks on the operating system dropdown to select an OS.
+
+![OS dropdown](img/Admin-create-4.6-0-1.jpg)
+
+The OS dropdown would promote OSs backed by images by placing them on top.
+It will also have a search field to fetch all desired results without going through two lists.
+This time, the user selects an OS without an available image/source.
+
+![No OS image available warning](img/Admin-create-4.6-0-2.jpg)
```

Comment:
The text for the action could be
```
Upload and attach an image for this operating system
```
as @glekner has pointed out - using the name of the OS can read strangely in cases where it's `centOS or higher`.

Reply:
Ok, @jelkosz FYI :)

Reply:
Should I drop the OS name for guest tools? as it is only windows.
@glekner 
Will leaving the "windows 10" help the user understand what are guest tools or is it common knowledge?

Reply:
I think what you have for the guest tools works fine. It doesn't hurt to provide that info. But good to have @glekner confirm too